### PR TITLE
Fix out of bounds error, print format type mismatch

### PR DESCRIPTION
**Pull request by svost**

### CODE REVIEW

File: src/agent/subagents/linux/drbd.cpp [hunk 1]
```diff
@@ -183,7 +183,7 @@ LONG H_DRBDDeviceList(const TCHAR *pszCmd, const TCHAR *pArg, StringList *pValue
 	{
 		if (s_devices[i].id != -1)
 		{
-			_sntprintf(szBuffer, 1024, _T("/dev/drbd%d %hs %hs/%hs %hs/%hs %hc"),
+                        _sntprintf(szBuffer, 1024, _T("/dev/drbd%d %hs %hs/%hs %hs/%hs %d"),
```
Comment (2128506) [inline src/agent/subagents/linux/drbd.cpp]: DRBD protocol indicated with single character, so %hc is correct  format here. %d will print that character's code instead.

Comment (svost) [reply src/agent/subagents/linux/drbd.cpp]: May be revert code to %hc and cast s_devices[i].protocol to proper type?

Comment (2128506) [reply src/agent/subagents/linux/drbd.cpp]: Yes, actually it could be %c and cast to TCHAR to avoid compiler warnings.

**Comment by svost** [comment]
Перезалил

**Pull request merged**